source/mathPres/MathCAT/MathCATPreferences.py

```diff
@@ -0,0 +1,1016 @@
+# -*- coding: UTF-8 -*-
+
+import math
```

Comment:
Can you run `isort` to sort the imports of these new files

`ruff check --select I --fix`